packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -79,6 +76,12 @@ export default {
     hasViewOptions: { type: Boolean, default: true },
     showActionsOnSelection: { type: Boolean, default: false }
   },
+  data: function () {
+    return {
+      resizeObserver: new ResizeObserver(this.onResize),
+      limitedScreenSpace: false
```

Comment:
Bit of an ugly name. I had set it to "showTooltips" originally, but the TopBar component should not determine wether to show or hide tooltips - that's up to the component itself. Therefore the TopBar only provides the info that limited screen space is available.

Reply:
Isn't the same valid for the `BatchActions` component then? That one still has the `showTooltips` prop.

Reply:
For `BatchActions`, there is no other condition to show tooltips. Hence I figured this component can have a `showTooltips` prop.